activiti-api-task-runtime/src/main/java/org/activiti/api/task/runtime/TaskAdminRuntime.java

```diff
@@ -74,5 +84,21 @@
 
     void setVariables(SetTaskVariablesPayload setTaskVariablesPayload);
 
-
+    /**
+     * Assign a task with a new user
+     *  - If the currently authenticated user is not a candidate throw an IllegalStateException
```

Comment:
@CTI777 here for the admin this doesn't apply, the admin can assign/reassign to anyone, that is why it is admin